stacks/patterns/index.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Title: Stacks
+ * Slug: stacks/index
+ * Categories: featured
+ */
+?>
+<!-- wp:group {"style":{"dimensions":{"minHeight":"100vh"},"position":{"type":"sticky","top":"0px"}},"backgroundColor":"white","layout":{"type":"flex","orientation":"vertical","verticalAlignment":"center","justifyContent":"center"}} -->
+<div class="wp-block-group has-white-background-color has-background" style="min-height:100vh"><!-- wp:group {"layout":{"type":"constrained"}} -->
+<div class="wp-block-group">
+
+<!-- wp:heading {"level":1,"fontSize":"x-large"} -->
+<h1 class="wp-block-heading has-x-large-font-size">Create Stacks.</h1>
```

Comment:
We should make all the strings translatable. I think @madhusudhand had a script for that somewhere? Could that be in a package we can use in this repo?

Reply:
That function lives [here](https://github.com/Automattic/themes/blob/trunk/theme-utils.mjs#L1213) and is used as an [npm script](https://github.com/Automattic/themes/blob/trunk/package.json#L31)

Maybe we could start a paired down version of `theme-utils` in this repo, and eventually extract the shared utilities into an npm package?

Reply:
I made a script for this here: https://github.com/WordPress/community-themes/pull/20

Reply:
Updated the strings